src/deposit/entities/deposit.entity.ts

```diff
@@ -1 +1,27 @@
-export class Deposit {}
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+
+@Schema()
+export class Deposit {
+  @Prop({
+    type: String,
+    required: true,
+  })
+  id: string;
+  @Prop({
+    type: String,
+    required: true,
+  })
+  idUser: string;
```

Comment:
popraw na userId

Reply:
jak byśmy chcieli byćdokładni to powinno być samo user